mani_skill/envs/tasks/tabletop/pull_drawer.py

```diff
@@ -0,0 +1,360 @@
+from typing import Dict, Union, Any
+import numpy as np
+import sapien
+import torch
+from mani_skill.agents.robots import Fetch, Panda
+from mani_skill.envs.sapien_env import BaseEnv
+from mani_skill.utils import sapien_utils
+from mani_skill.utils.registration import register_env
+from mani_skill.utils.scene_builder.table import TableSceneBuilder
+from mani_skill.utils.structs.types import SimConfig, GPUMemoryConfig
+from mani_skill.sensors.camera import CameraConfig
+from mani_skill.utils.structs import Pose
+from mani_skill.utils.building import actors
+
+@register_env("PullDrawer-v1", max_episode_steps=200)
+class PullDrawerEnv(BaseEnv):
```

Comment:
we recently updated how we document tasks/environments. It is nearly completely automatically generated now. Can you follow the documentation style of e.g. PushCube? Moreover can you save an example video with ray tracing of this task and put it in the folder with the others?